zerobuf/Zerobuf.cpp

```diff
@@ -30,12 +30,12 @@ Zerobuf::Zerobuf( AllocatorPtr alloc )
 
 Zerobuf::Zerobuf( Zerobuf&& rhs )
 {
-    rhs.notifyChanging();
     _allocator = std::move( rhs._allocator );
     rhs._allocator.reset( new NonMovingAllocator( rhs.getZerobufStaticSize(),
                                                  rhs.getZerobufNumDynamics( )));
     uint32_t& version = rhs._allocator->getItem< uint32_t >( 0 );
     version = ZEROBUF_VERSION_ABI;
+    rhs.notifyUpdated();
```

Comment:
Agree, it should be removed. For the constructor (this->)notifyChanging() has no effect in any case because nothing can be connected to it already.